Detections/SecurityEvent/RDP_MultipleConnectionsFromSingleSystem.yaml

```diff
@@ -0,0 +1,30 @@
+id: 78422ef2-62bf-48ca-9bab-72c69818a425
+name: Multiple RDP connections from Single System
+description: |
+  'Identifies when an RDP connection is made to multiple systems from the same system with the same account within the same day.  
+  RDP connections are indicated by the logged EventID 4624 with LogonType = 10'
+severity: Low
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - LateralMovement
+relevantTechniques:
+  - T1076
+query: |
+
+  let timeframe = 1d;
+  // The threshold below includes matching on computer counts by account higher than 3 in a given day.  Change the threshold as needed.
+  let threshold = 3;
```

Comment:
Given this figure will vary a lot per environment could this be more effectively implemented by doing some baselining? Say if the number of connections is higher than the average in the last week?

Reply:
Done